packages/common/container-definitions/src/loader.ts

```diff
@@ -369,10 +369,20 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 	 * Closes the container and returns serialized local state intended to be
 	 * given to a newly loaded container.
 	 * @experimental
-	 * {@link https://github.com/microsoft/FluidFramework/packages/tree/main/loader/container-loader/closeAndGetPendingLocalState.md}
+	 * {@link https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/closeAndGetPendingLocalState.md}
 	 */
 	closeAndGetPendingLocalState(): string;
 
+	/**
+	 * Get pending state from container. WARNING: misuse of this API can result in duplicate op
+	 * submission and potential document corruption. The blob returned MUST be deleted if and when this
+	 * container emits a "connected" event.
+	 * @returns serialized blob that can be passed to Loader.resolve()
+	 * @experimental misuse of this API can result in duplicate op submission and potential document corruption
+	 * {@link https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/closeAndGetPendingLocalState.md}
+	 */
+	getPendingLocalState(): string;
```

Comment:
in general i'd like to move away from adding experimental features to existing interfaces. it makes it difficult to manage breaking changes, and doesn't require distinct awareness from the consumer to take a dependency on such experimental features, specifically intellisense will show this by default, making it easy to accidentally use. 


For this PR i can see a few options:
- create new experimental interface in container loader package which requires and explicit cast from IContainer to something like IContainerExperimentalOffline to get access to the method
- in container loader only expose a function, getPendingLocalStateExperimental(container: IContainer) which internally cast the provide IContainer to Container to call the real methods.